CHANGES.rst

```diff
@@ -4,6 +4,11 @@
 Bug Fixes
 ---------
 
+- Fixed bug where subsets applied with remove / andNot mode when wcs linked were not able to return
+  sky regions. [#3547]
+
+- Fixed bug on MOSVIZ where an exception was raised when loading JWST S2D file from a directory. [#3570]
+
 - Fixed viewer layer to persist when changing jupyter/browser tabs. [#3551]
```

Comment:
Woops, can you fix this in this PR and I'll sneak the same fix into main in a separate open PR there?

```suggestion
- Fixed viewer layout to persist when changing jupyter/browser tabs. [#3551]
```